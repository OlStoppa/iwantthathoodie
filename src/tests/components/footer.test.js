import React from 'react'
import { render } from '@testing-library/react'
import Footer from '../../components/footer'


test('renders', () => {
    const { asFragment } = render(<Footer/>);
    expect( asFragment()).toMatchSnapshot();
});