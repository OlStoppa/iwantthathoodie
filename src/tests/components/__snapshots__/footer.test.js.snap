// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <div
      class="footer--inner"
    >
      Â© 2019 iwantthathoodie.com
    </div>
  </div>
</DocumentFragment>
`;
